### PR TITLE
feat: [GTM-781]: update background img

### PR DESCRIPTION
**Pull request by nanaxuHarness**
this is to update sign in/up img

saas:

https://user-images.githubusercontent.com/80717697/158896117-a7240dab-8650-4b23-a774-8541f5c53d8d.mov

community:

https://user-images.githubusercontent.com/80717697/158896045-eef83894-294b-4eb6-83f6-156964ea756f.mov

### CODE REVIEW

File: src/components/BasicLayout/BasicLayout.tsx [hunk 1]
```diff
@@ -25,6 +26,9 @@ const BasicLayout: React.FC<BasicLayoutProps> = ({ children }) => {
       </div>
       <div className={css.imageColumn}>
         <img className={css.image} src={bgImg} alt="" aria-hidden />
+        {!isCommunityPlan() && (
```
Comment (abhinavrastogi-harness) [inline src/components/BasicLayout/BasicLayout.tsx:29]: what do we show in community plan? blank screen on the right with just the dots?

**Review by abhinavrastogi-harness — COMMENTED** [review]
we should try tiling the dots background instead of scaling it to 100%

File: src/components/BasicLayout/BasicLayout.tsx (hunk 1 above)
Comment (nanaxuHarness) [reply src/components/BasicLayout/BasicLayout.tsx]: updated

**Comment by nanaxuHarness** [comment]
> we should try tiling the dots background instead of scaling it to 100%

updated, separate the bg dots as `repeat` to scale it

**Review by tan-nhu — APPROVED** [review]

**Review by allenthom — APPROVED** [review]

**Pull request merged**